src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/SupportPages._ViewStart.cshtml

```diff
@@ -1,4 +0,0 @@
-@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase
-@@{
-    Layout = "@(Model.LayoutPageNoExtension)";
```

Comment:
Should we remove the associated Layout files as well? 

- https://github.com/dotnet/Scaffolding/blob/main/src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/_Layout.cshtml
- https://github.com/dotnet/Scaffolding/blob/main/src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap4/Pages/_Layout.cshtml
- https://github.com/dotnet/Scaffolding/blob/main/src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap5/Pages/_Layout.cshtml